docs/lessons/random.rst

```diff
@@ -0,0 +1,88 @@
+Random
+------
+
+Sometimes you want to leave things to chance, or mix it up a little. You want
+the device to act randomly.
+
+MicroPython comes with a ``random`` module to make it easy to introduce chance
+and a little chaos into your code. For example, here's how to scroll a random
+name across the display::
+
+    from microbit import *
+    import random
+
+    names = ["Amy", "Ben", "Damien", "Alia", "Kushal", "Sophie", "Zoltan" ]
+
+    display.scroll(random.choice(names))
+
+The list (``names``) contains seven names defined as strings of characters.
+Using the "onion" effect introduced when discussing *nesting* the
+``random.choice`` method takes this list as an argument and returns an item
+chosen at random. This item (the randomly chosen name) is the argument for
+``display.scroll``.
+
+Random Numbers
+++++++++++++++
+
+Random numbers are very useful. They're common in games. Why else do we have
+dice?
```

Comment:
On 18/02/2016 12:24, Carlos wrote:
> In docs/lessons/random.rst
> <https://github.com/bbcmicrobit/micropython/pull/212#discussion_r53306783>:
> 
>> +
>> +    names = ["Amy", "Ben", "Damien", "Alia", "Kushal", "Sophie", "Zoltan" ]
>> +
>> +    display.scroll(random.choice(names))
>> +
>> +The list (``names``) contains seven names defined as strings of characters.
>> +Using the "onion" effect introduced when discussing *nesting* the
>> +``random.choice`` method takes this list as an argument and returns an item
>> +chosen at random. This item (the randomly chosen name) is the argument for
>> +``display.scroll``.
>> +
>> +Random Numbers
>> +++++++++++++++
>> +
>> +Random numbers are very useful. They're common in games. Why else do we have
>> +dice?
> 
> should it be |dices|?

Nope. The singular (seldom used) is die. The plural is dice.